deployment/src/main/java/io/quarkiverse/jasperreports/deployment/ReportRootsBuildItem.java

```diff
@@ -0,0 +1,28 @@
+package io.quarkiverse.jasperreports.deployment;
+
+import java.util.Iterator;
+import java.util.Set;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+
+/**
+ * The set of report root paths.
+ */
+public final class ReportRootsBuildItem extends SimpleBuildItem implements Iterable<String> {
```

Comment:
This class doesn't seem to be useful, you can consume `List<ReportRootBuildItem>` instead?

Reply:
I was following the code from the Qute extension, which has it set up this way.  I can give it a go with simplifying it though.

Reply:
Will be fixed in the next push.